### PR TITLE
Added svg support

### PR DESCRIPTION
**Pull request by chenlevy24**
Changed `ImageHelper` and `util` to support svg type images.

### CODE REVIEW

File: packages/core/src/lib/wrappers/svg2img.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const _svg2img = require('svg2img');
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Callback = (err: any, buffer: any) => any;
+
+export enum Format {
```
Comment (PEConn) [inline packages/core/src/lib/wrappers/svg2img.ts:7]: Do you mind adding a comment somewhere in this class that they're only reason we're doing this is that we couldn't get their types to work, and that if we do in the future, we can remove most of this.

**Review by PEConn — APPROVED** [review]
This is looking great, thanks!

**Review by andreban — CHANGES_REQUESTED** [review]
Generally looks good. Dropped some notes.

File: packages/core/src/lib/wrappers/svg2img.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+
+// This class is needed because svg2img's typescript type definitions didn't work.
+// once it will, there is no need for this class.
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const _svg2img = require('svg2img');
```
[the remark this answers is not in the document]
Comment (chenlevy24) [reply packages/core/src/lib/wrappers/svg2img.ts:21]: it isn't working

File: packages/core/src/lib/wrappers/svg2img.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Callback = (err: any, buffer: any) => any;
```
[the remark this answers is not in the document]
Comment (chenlevy24) [reply packages/core/src/lib/wrappers/svg2img.ts]: I just removed it, it being used in only one place so there is no need for a special type

**Review by andreban — APPROVED** [review]
LGTM, Thank you!

**Comment by JudahGabriel** [comment]
Is this available in a stable Bubblewrap release?

**Comment by andreban** [comment]
This has been published on 1.7.0 (https://github.com/GoogleChromeLabs/bubblewrap/releases/tag/v1.7.0)

**Pull request merged**